__tests__/pet.test.js

```diff
@@ -0,0 +1,179 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+      expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+    it ('sets the name property', () => {
+      const pet = new Pet ('Prince')
+      expect(pet.name).toEqual('Prince');
+    });
+    it ('has an initial age of 0', () => {
+      const pet = new Pet ('Prince')
+      expect(pet.age).toEqual(0);
+    });
+    it ('has an initial hunger of 0', () => {
+      const pet = new Pet ('Prince')
+      expect(pet.hunger).toEqual(0);
+    });
+    it ('has an initial fitness of 10', () => {
+      const pet = new Pet ('Prince')
+      expect(pet.fitness).toEqual(10);
+
+    });
+  });
+
+  describe('isAlive', () => {
+    it('tells you if your pets fitness is 0 so has died', () => {
+
```

Comment:
You dont need to add a clear. line at the start of each of your test blocks, this could be removed